services-js/commissions-search/src/FacetList.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from "react";
+import "./FacetList.css";
+
+import gql from "graphql-tag";
+import { Query } from "react-apollo";
+
+class FacetList extends Component {
+  render() {
+    const GET_AREAS = gql`
+      {
+        policyTypes {
+          id
+          name
+        }
+      }
+    `;
+
+    function Area(props) {
+      return (
+        <label className="cb">
+          <input
+            id="checkbox-area"
+            name={props.id}
+            type="checkbox"
+            value={props.areaValue}
+            className="cb-f"
+            checked={props.areaValue}
+            onChange={props.handleCheckChange}
+          />
+          <span className="cb-l">{props.name}</span>
+        </label>
+      );
+    }
+
+    const PolicyAreaComponents = () => (
+      <Query query={GET_AREAS}>
+        {({ loading, error, data }) => {
+          if (loading) return "Loading...";
+          if (error) return `Error! ${error.message}`;
+
+          return data.policyTypes.map(policy_area => (
```

Comment:
`policyArea`

Reply:
@finneganh what's the issue here?

Reply:
JavaScript variables should be camelCase